apps/site/public/types/modules/graph/0.3/schema/aggregation-operation.json

```diff
@@ -1,87 +1,125 @@
+                  {
+                    "type": "object",
+                    "properties": {
+                      "field": {
+                        "$ref": "#/$defs/Field"
+                      },
+                      "operator": {
+                        "$ref": "#/$defs/FilterOperatorRequiringValue"
+                      },
+                      "value": {
+                      }
+                    },
+                    "required": [
+                      "field",
+                      "operator",
+                      "value"
+                    ],
+                    "additionalProperties": false
                   },
-                  "value": {
-                    "type": "string"
+                  {
+                    "type": "object",
+                    "properties": {
+                      "field": {
+                        "$ref": "#/$defs/Field"
+                      },
+                      "operator": {
+                        "$ref": "#/$defs/FilterOperatorWithoutValue"
+                      }
+                    },
+                    "required": [
+                      "field",
+                      "operator"
+                    ],
+                    "additionalProperties": false
                   }
-                },
-                "type": "object"
+                ]
+              }
+            },
+            "operator": {
+              "$ref": "#/$defs/MultiFilterOperatorType"
+            }
+          },
+          "required": [
+            "filters",
+            "operator"
+          ],
+          "additionalProperties": false
+        },
+        "multiSort": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "field": {
+                "$ref": "#/$defs/Field"
               },
-              {
-                "properties": {
-                  "field": {
-                    "type": "string"
+              "desc": {
+                "oneOf": [
+                  {
+                    "type": "boolean"
                   },
-                  "operator": {
-                    "$ref": "#/$defs/FilterOperatorWithoutValue"
+                  {
+                    "type": "null"
                   }
-                },
-                "type": "object"
+                ]
               }
-            ]
-          },
-          "type": "array"
-        },
-        "operator": {
-          "$ref": "#/$defs/MultiFilterOperatorType"
+            },
+            "required": [
+              "field"
+            ],
+            "additionalProperties": false
+          }
         }
       },
-      "type": "object"
+      "additionalProperties": false
     },
-    "multiSort": {
+    "graphResolveDepths": {
+      "$ref": "https://blockprotocol.org/types/modules/graph/0.3/schema/partial-graph-resolve-depths"
+    }
+  },
+  "required": [
+    "operation"
+  ],
+  "additionalProperties": false,
```

Comment:
I think we can remove all of these. To my knowledge we aren't treating it as an error if people send more fields in the messages, we just ignore them.

Reply:
Commit, which removes the majority of these: [Remove additionalProperties where not needed](https://github.com/blockprotocol/blockprotocol/pull/1034/commits/470035c00d8995d6754600e88f9477a12240034a)